kafkabp/prometheus.go

```diff
@@ -0,0 +1,65 @@
+package kafkabp
+
+import (
+	"github.com/prometheus/client_golang/prometheus"
+	"github.com/prometheus/client_golang/prometheus/promauto"
+)
+
+const (
+	promNamespace = "kafkabp"
+
+	subsystemConsumer      = "consumer"
+	subsystemGroupConsumer = "group_consumer"
+
+	successLabel = "kafka_success"
```

Comment:
I think @SuperQ indicated that for most cases a prefix is not necessary

Reply:
Yea, I'm not sure about this one. On one hand, it's a bit redundant. But it's also following the pattern of other systems. It also separates the index, which might be a little faster in the inverted index lookups.

Reply:
I think even though adding prefix is not *needed* in most cases, since it's still needed for some cases (to avoid conflicting against some external labels), it's easier to just always prefix them so people don't have to figure out "is this label an external label I need to prefix". The potential performance benefit just reinforces the "always prefix" practice :)